internal/gw/client.go

```diff
@@ -23,6 +23,14 @@ import (
 	"github.com/release-engineering/exodus-rsync/internal/walk"
 )
 
+func logConnectionOpen(ctx context.Context, url string) {
+	log.FromContext(ctx).Infof("Initializing connection to %s", url)
```

Comment:
It seems like this should be using the field-based logging like most other log statements do, e.g.

```suggestion
	log.FromContext(ctx).F("url", url).Info("Initializing connection")
```

Shouldn't it?  Then in Splunk or elsewhere we will be able to search for all log messages having 'url' field equal to a specific value, rather than having to handle it by a custom regex like "Initialization connection to (?<url>.*)" .

Reply:
Ah, yeah, I do prefer that. Thanks!